Makefile

```diff
@@ -15,8 +15,8 @@
 DOCKER_ARCHS ?= amd64 armv7 arm64
 
 TSDB_PROJECT_DIR = "./tsdb"
-TSDB_CLI_DIR="$(TSDB_PROJECT_DIR)/cmd/tsdb"
-TSDB_BIN = "$(TSDB_CLI_DIR)/tsdb"
+TSDB_CLI_DIR = "./cmd/promtool"
+TSDB_BIN = "$(TSDB_CLI_DIR)/promtool"
```

Comment:
Why not use `./promtool` instead of rebuilding the binary?

Reply:
Done